config/base.keymap

```diff
@@ -8,11 +8,47 @@
 #include <dt-bindings/zmk/bt.h>
 #include <dt-bindings/zmk/keys.h>
 
+// Layer names
 #define QWERTY_L 0
 #define NAV_L 1
 #define SYM_L 2
 #define FUN_L 3
 
+// For Home row mods on a 36 key board
+#include "zmk-helpers/helper.h"
+#include "zmk-helpers/key-labels/36.h"                                     // key-position labels
+#define KEYS_L LT0 LT1 LT2 LT3 LT4 LM0 LM1 LM2 LM3 LM4 LB0 LB1 LB2 LB3 LB4  // left-hand keys
+#define KEYS_R RT0 RT1 RT2 RT3 RT4 RM0 RM1 RM2 RM3 RM4 RB0 RB1 RB2 RB3 RB4  // right-hand keys
+#define THUMBS LH2 LH1 LH0 RH0 RH1 RH2                                      // thumb keys
+
+/* left-hand HRMs, e.g. `&hml LSHIFT F` */
+ZMK_HOLD_TAP(hml,
+    flavor = "balanced";
+    tapping-term-ms = <280>;
+    quick-tap-ms = <175>;                // repeat on tap-into-hold
+    require-prior-idle-ms = <150>;
+    bindings = <&kp>, <&kp>;
+    hold-trigger-key-positions = <KEYS_R THUMBS>;
+    hold-trigger-on-release;             // delay positional check until key-release
+)
+
+/* right-hand HRMs, e.g. `&hmr LSHIFT J` */
+ZMK_HOLD_TAP(hmr,
+    flavor = "balanced";
+    tapping-term-ms = <280>;
+    quick-tap-ms = <175>;                // repeat on tap-into-hold
+    require-prior-idle-ms = <150>;
+    bindings = <&kp>, <&kp>;
+    hold-trigger-key-positions = <KEYS_L THUMBS>;
+    hold-trigger-on-release;             // delay positional check until key-release
+)
+
+/**
+Home row mods will be CAGS, Control Alt Gui Shift on the left hand side
+                      SGAC, inverted on the left
+Shift in center, then GUI, Alt under the ring finger, and Control under the pinky
```

Comment:
Let's go ahead and do SCAG with shift on the pinkies. 

> The Shift modifier is unlike all the others. Unless it’s paired with another modifier, it feels more like a layer switch than a key we can use to execute keyboard shortcuts.

We'll keep using thumb shift for normal typing, and home row shift for shortcuts

Reply:
While we're at it, we could try urob's shift thumb stuff:

- tap is a single letter shift
- hold is regular shift
- double-tap is CAPS-WORD

https://github.com/urob/zmk-config?tab=readme-ov-file#capsword